pages/contact.tsx

```diff
@@ -0,0 +1,55 @@
+import { FC } from 'react'
+import { GetStaticProps } from 'next'
+import { useTranslation } from 'next-i18next'
+import { serverSideTranslations } from 'next-i18next/serverSideTranslations'
+import Layout from '../components/Layout'
+import LinkButton from '../components/LinkButton'
+
+const Contact: FC = () => {
+  const { t } = useTranslation('contact')
+
+  return (
+    <Layout
+      meta={t('common:meta', { returnObjects: true })}
+      header={t('common:header', { returnObjects: true })}
+      footer={t('common:footer', { returnObjects: true })}
+    >
+      <h1 className="mb-4">{t('header')}</h1>
+      <h2 className="my-14">{t('description')}</h2>
+      <div className="space-y-4 text-xl">
+        <div id="contactUs">
+          <LinkButton
+            href={t('links.contact.link')}
+            text={t('links.contact.text')}
+            external
+          ></LinkButton>
+        </div>
+        <div id="findServiceLocation">
+          <LinkButton
+            href={t('links.findServiceLocation.link')}
+            text={t('links.findServiceLocation.text')}
+            external
+          ></LinkButton>
+        </div>
+        <div id="bookAppointment">
+          <LinkButton
+            href={t('links.bookAppointment.link')}
+            text={t('links.bookAppointment.text')}
+            external
+          ></LinkButton>
+        </div>
```

Comment:
I'm pretty sure these are the same links found on our other pages like the status results (although maybe one if the links isn't right) Wondering if it makes sense to re-use the component (you do have it styled differently, so a class(es) could be passed into the component)
https://github.com/DTS-STN/passport-status/blob/main/pages/status.tsx#L131
Do note, that I'm suggesting a change to how the text for it is extracted from common https://github.com/DTS-STN/passport-status/pull/85/files#diff-9679a3f1633cdb4d4062246a217d7eb49bdcc6b5889b090e82e1ddc06dc76518R79

Reply:
No issue switching over to using the LinkSummary component, I also agree with your second point/change.